libqtile/config.py

```diff
@@ -82,14 +82,14 @@ class KeyChord:
         current chord/mode).
     """
     def __init__(self, modifiers: List[str], key: str,
-                 submapings: List[Union[Key, KeyChord]], mode: str = ""):
+                 submappings: List[Union[Key, KeyChord]], mode: str = ""):
```

Comment:
When was the spelling error introduced? As this is part of the config API, unless it was introduced super recently we should probably add a changelog and `qtile migrate` entry for this change.

Reply:
Didn't think about that, it might be a good idea. Although I doubt that the literally like 3 people that already started using `KeyChord`s since they became available recently instantiate the `KeyChord` objects with `submapings=[...]` explicitly.
But your point is completely fine, I have to find out how that `migrate` thing works under the bonnet.

Reply:
The commit that fixes this typo is the very last making it easy to `commit --amend` if the migrate thing is decided to be worth it. Leaving this unresolved for further opinions.